### PR TITLE
[doc  ] Add vouchered offer tutorial, clean up tutorial listing

### PR DESCRIPTION
**Pull request by tshuck**
@hatchloyalty/core @natashajanic 

Adds a tutorial for creating vouchered offers, also cleans up whitespace in the guides/tutorials list page a bit while we figure out some better organization:
tutorial-list

### CODE REVIEW

File: content/static-pages/tutorials/behavioral-offer.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Learn how to translate a non-transactional, behavioral marketing initiative in t
...
+Hatch provides powerful tools to strengthen personal relationships with a loyalty program's Membership base. These relationships lead to a beneficial change in consumer behavior: a Membership is engaged with offerings that they are personally interested in, and the loyalty program can incentivize opportunities to get to know a Membership even more. Just as a [transactional offer](/tutorials/transactional-offer) may be configured to support a wide range of purchased-based behaviors, Hatch also supports a wide range of non-purchase, non-transactional Offers. Hatch uses [Loyalty Events](/api#loyalty-events) to capture all types of non-transactional activity.
```
Comment (tshuck) [inline content/static-pages/tutorials/behavioral-offer.md]: Github doesn't highlight the change, but it's just a typo fix from `Memberhship` -> `Membership` on the first line.  Same deal with the update to a similar paragraph in the transactional-offer tutorial.

File: content/static-pages/tutorials/vouchered-offer.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+1.  [Allow a member to use their vouchers](#step-5-allow-a-member-to-use-their-vouchers)
+    1.  [Query for available vouchers at the point-of-sale](#step-51-query-for-vouchers-at-the-point-of-sale)
+    1.  [Apply the discount](#step-52-apply-the-discount)
+    1.  [Record used vouchers](#step-53-record-used-vouchers)
+
+## Steps
+
+### Step 1: _Describe the material benefit_
+
+A material benefit may take many forms: an entry in a sweepstakes, or admittance to a personalize event (e.g., a "thank you" outing for highly-engaged members). In this case, the benefit will be a discount on a product purchase.
```
Comment (camneu37) [inline content/static-pages/tutorials/vouchered-offer.md]: typo sentence one, change:
"...or admittance to a personalize event"
to:
"...or admittance to a personalize**d** event"

File: content/static-pages/tutorials/vouchered-offer.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+> Elsewhere, we'll set up an offer for coffee-club members to [purchase the discount](/tutorials/purchasable-offers) in exchange for points, and we'll also incentivize members that usually [only buy gas](/tutorials/voucher-award) to come in with a limited number of vouchers.
+
+Now that we've described the material benefit and have an idea for how it will be granted, we may begin configuring it in Hatch.
+
+### Step 2: _Create the Offer_
+
+Hatch represents a marketing initiative via the [Offer](/guides/offer). The Offer serves as a central connection upon which its various components are built. From our initial program set-up, we know that our loyalty program is identified by the chain_id **e6cee754-faf8-46a4-99fd-47ba6c3201bc**. We'll use this to make sure that the Offer goes live in our loyalty program. We'll configure a few more attributes of the Offer as well:
+
+- Offer to be published on May 31st, and will end on September 30th. We do this by setting `publish_at` to "20xx-05-31" and `expire_at` to "20xx-09-30".
+- The Offer will be referenced internally by setting the `name` to "Discounted Coffee" Memberships do not see this title; it's only used by our systems.
```
Comment (camneu37) [inline content/static-pages/tutorials/vouchered-offer.md]: typo - needs a period between "Discounted Coffee" and Memberships

File: content/static-pages/tutorials/vouchered-offer.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+We can inspect the response code to ensure that the Earning Mechanism was created successfully (code 201), but we don't need any of the rest of the Earning Mechanism data in order to continue.
+
+### Step 4: _Write and upload marketing copy_
+
+Since member targeting (via [Audiences](/guides/offers/audiences) and limitation (via [Limits](/guides/offers/limits)) are configured on the offers that _grant_ vouchers, the last component to create for the vouchered offer is
+the [Marketing Contents](/guides/offers/marketing-contents) that will display consumer-facing copy to the member. Marketing contents are created via the [create marketing contents](/api#create-marketing-content) api endpoint, and might consist of the following:
+
+```shell
+curl -X POST \
+  https://api.hatchloyalty.com/api/v2/product_groups \
```
Comment (camneu37) [inline content/static-pages/tutorials/vouchered-offer.md]: need to change this to marketing contents endpoint

File: content/static-pages/tutorials/vouchered-offer.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+
+#### Step 5.2: _Apply the discount_
+
+Since the point-of-sale is responsible for applying discounts to a member's purchase, it must have some configuration that allows it to recognize the purchase of the cold-brew coffee; it may also use the offer's `external_id` to link its discount-configuration to Hatch's understanding of how many vouchers are available for that member.
+
+In this case, the member has **4** available vouchers for half-off cold-brew. In our example, the member purchases **2** cold-brew coffees, and the point-of-sale provides the half-off discount on each.
+
+#### Step 5.3: _Record used vouchers_
+
+The member has purchased **2** cold-brew coffees, received **2** discounts, and as such has used **2** of their available vouchers. Because this _vouchered offer_ is configured with the _External Fulfillment_ Earning Mechanism, the point-of-sale will indicate to Hatch that the offer has been fulfilled twice by recording the offer's id (**2b591c20-0f25-4e47-bf02-d82d784d983c** in the resultant [Transaction](/api/#create-transaction)'s `fulfilled_offer_ids` field **twice**.
```
Comment (camneu37) [inline content/static-pages/tutorials/vouchered-offer.md]: Missing closing parenthesis after the offer's ID

File: content/static-pages/tutorials/vouchered-offer.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+        "group": "coffee"
+      } ],
+      "membership_id": "354067ec-ff43-487b-91ed-9134ac79954d",
+      "fulfilled_offer_ids": ["2b591c20-0f25-4e47-bf02-d82d784d983c", "2b591c20-0f25-4e47-bf02-d82d784d983c"]
+    }
+  }
+}'
+```
+
+Once the transaction has been processed by Hatch, two of the offer's associated vouchers will be marked as "fulfilled", a future calls to the Membership-Offer will show **2** available vouchers in this offer's `voucher_count`.
```
Comment (camneu37) [inline content/static-pages/tutorials/vouchered-offer.md]: typo, change:
"...'fulfilled', a future calls to the Membership-Offer"
to:
"...'fulfilled', **and** future calls to the Membership-Offer"

**Pull request merged**